scripts/config.pl

```diff
@@ -129,7 +130,7 @@
 
 # Things that should be enabled in "full" even if they match @excluded
 my @non_excluded = qw(
-PLATFORM_[A-Z0-9]+_ALT
+PLATFORM_(?!ZEROIZE_)[A-Z0-9]+_ALT
```

Comment:
The commit message doesn't adequately explain why we mustn't exclude MBEDTLS_PLATFORM_ZEROIZE_ALT from being excluded. Could you add the reasoning behind the change to the commit message? In particular, why would we want to prevent zeroize from being included in full?

Reply:
I added the reasoning in a comment in a subsequent commit. I'm not going to rewrite the old commit because it adhered to the standard for documentation at the time.